frontend/src/metabase-types/store/entities.ts

```diff
@@ -0,0 +1,7 @@
+import { Database } from "metabase-types/types/Database";
```

Comment:
These are wrong types! All types from `/types` are deprecated and should not be used. Please use types from '/api`.